flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -1736,7 +1736,7 @@ object TaskManager {
     : (String, java.util.Iterator[Integer]) = {
 
     var taskManagerHostname = configuration.getString(
-      ConfigConstants.TASK_MANAGER_HOSTNAME_KEY, null)
+      TaskManagerOptions.HOST, null)
```

Comment:
Maybe we can make it simple for `configuration.getString(TaskManagerOptions.HOST)` directly.